### DISCUSSION TITLE
leafmap.write_lidar(las, 'madison.las')  AttributeError: module 'leafmap' has no attribute 'write_lidar'

### DISCUSSION THREAD

**Discussion by lxy1888666** [opengeos/leafmap#280] [Q&A] [answered]
When I use examples 55_lidar.ipynb, it orrurs an error :
AttributeError  Traceback (most recent call last)
Input In [14], in <cell line: 1>()
----> 1 leafmap.write_lidar(las, 'madison.las')
AttributeError: module 'leafmap' has no attribute 'write_lidar'

I wonder if anyone has the same problem like this and what is the correct function of "leafmap,write_lidar"

The "convert_lidar" has the same problem
AttributeError     Traceback (most recent call last)
Input In [18], in <cell line: 1>()
----> 1 las = leafmap.convert_lidar(laz, file_version='1.4')
AttributeError: module 'leafmap' has no attribute 'convert_lidar'

The "lasp[lazrs]' function has been successfully installed,so I really don't understand why the problem occured

**Comment by giswqs** [comment, marked as answer]
Make sure you update leafmap to the latest version: `pip install -U leafmap`

**Reply by lxy1888666** [reply]
The version of leafmap in my computer is 0.10.3. But it doesn't work. Besides, if I use planetary computer, is it necessary to update le package?
